### PR TITLE
rpc: Remove pubkey from getBalance response

### PR DESCRIPTION
**Pull request by mvines**

### CODE REVIEW

File: src/rpc.rs [hunk 1]
```diff
@@ -106,7 +106,7 @@ impl RpcSol for RpcSolImpl {
         let signature = Signature::new(&signature_vec);
         meta.request_processor.get_signature_status(signature)
     }
-    fn get_balance(&self, meta: Self::Metadata, id: String) -> Result<(String, i64)> {
+    fn get_balance(&self, meta: Self::Metadata, id: String) -> Result<i64> {
         let pubkey_vec = match bs58::decode(id).into_vec() {
```
Comment (garious) [inline src/rpc.rs]: While you're in the neighborhood, `map_err(|_| Error::invalid_request())`

And delete commented out code below?

File: src/rpc.rs [hunk 2]
```diff
@@ -157,9 +157,9 @@ impl JsonRpcRequestProcessor {
     }
 
     /// Process JSON-RPC request items sent via JSON-RPC.
-    fn get_balance(&self, pubkey: Pubkey) -> Result<(String, i64)> {
+    fn get_balance(&self, pubkey: Pubkey) -> Result<i64> {
```
Comment (garious) [inline src/rpc.rs]: Does that `Result` wrapper add value?

File: src/rpc.rs (hunk 1 above)
Comment (mvines) [reply src/rpc.rs]: > And delete commented out code below?

You mean `send_transaction`?

Comment (garious) [reply src/rpc.rs]: Oh yeah, I didn't click expand enough times in the github UI. All good.

File: src/rpc.rs (hunk 2 above)
Comment (mvines) [reply src/rpc.rs]: Looks like we either wrap here or in the caller, RpcSolImpl.  Seems about even either way

File: src/rpc.rs (hunk 1 above)
Comment (mvines) [reply src/rpc.rs]: `map_err` applied, much nicer

**Pull request merged**